en/02_Developer_Guides/00_Model/01_Data_Model_and_ORM.md

```diff
@@ -28,6 +28,8 @@ use SilverStripe\ORM\DataObject;
 
 class Player extends DataObject
 {
+    private static $class_description = 'Sports ball player';
+
```

Comment:
This is theoretically the first docs that show how to create a `DataObject` class so I'm adding best practice here.

I'm not adding it to _all_ docs that show creating a `DataObject` class because a) that's a _lot_ of examples and b) it's not really relevant for most of them.